terraform/environment/staging.backend.conf

```diff
@@ -0,0 +1,4 @@
+bucket  = "shift3-terraform-state"
+key     = "test-staging-boilerplate-client-react/terraform.tfstate"
+region  = "us-west-2"
+profile = "shift3"
```

Comment:
I think `shift3-super` is the new default, let's swap this out, update the merge conflicts and this should be good to merge!